index.js

```diff
@@ -110,6 +111,10 @@ class RenderPDF {
     }
 
     async renderPdf(url, options) {
+        if (!url.match(/^(http|https|file):\/\//)) {
```

Comment:
Could you change it to `/^(https?|file):\/\//`
and before passing that to `file-url` (maybe we could not introduce another dependency?) we need to check if file exists on local filesystem.
Currently passing something like `www.example.com` works, but after this PR this would fail and I would like to avoid introducing breaking changes

Reply:
@Szpadel I believe I have all the changes you requested pushed.  

Only thing I'm not sure about is you mentioned that currently passing `www.example.com` works.  Are you saying it should work without adding `https://` to the front?  I tried that on the current version (without my changes) and it doesn't appear to work for me.  I'm on my Windows machine right now though so I don't know if maybe it works on other platforms.  I can check on my Mac in a little bit.

Reply:
@Szpadel anything else you need me to do for this one?